packages/yourloops/test/integration/pages/notification/notification-for-patient.spec.tsx

```diff
@@ -0,0 +1,126 @@
+import { mockPatientLogin } from '../../mock/patient-login.mock'
+import { monitoredPatientAsTeamMember } from '../../mock/patient.api.mock'
+import NotificationApi from '../../../../lib/notifications/notification.api'
+import { Notification } from '../../../../lib/notifications/models/notification.model'
+import { TeamMemberRole } from '../../../../lib/team/models/enums/team-member-role.enum'
+import { NotificationType } from '../../../../lib/notifications/models/enums/notification-type.enum'
+import { invitationHcpForPatient, teamOne, teamTwo } from '../../mock/team.api.mock'
+import TeamAPI from '../../../../lib/team/team.api'
+
+describe('Add patient in the team with a notification', () => {
+  const invitationTeam: Notification = {
+    id: '26a11710e98e4',
+    type: NotificationType.careTeamPatientInvitation,
+    metricsType: 'join_team',
+    email: 'ylp.ui.test.67-patient12@diabeloop.fr',
+    creatorId: 'dec0816009d32',
+    date: '',
+    target: { id: '63c7b7989cacc878ecce2c40', name: 'sysReq-67-team2' },
+    role: TeamMemberRole.patient,
+    creator: {
+      userid: 'dec0816009d32',
+      profile: {
+        fullName: 'Backloops test SysReq 67 hcp1',
+        patient: {
+          birthday: '',
+          diagnosisDate: ''
+        },
+        email: ''
+      }
+    }
+  }
+
+  beforeAll(() => {
+    mockPatientLogin(monitoredPatientAsTeamMember)
+    jest.spyOn(TeamAPI, 'getTeams').mockResolvedValue([teamOne, teamTwo])
+    jest.spyOn(NotificationApi, 'getReceivedInvitations').mockResolvedValue([invitationTeam])
+    jest.spyOn(TeamAPI, 'getTeamFromCode').mockResolvedValue(invitationHcpForPatient)
+    jest.spyOn(NotificationApi, 'acceptInvitation').mockResolvedValue()
+  })
+
+  it('Should cancel add team', async () => {
+    await act(async () => {
+      renderPage('/notifications')
+    })
+
+    expect(screen.getByTestId('notification-line')).toHaveTextContent("You're invited to share your diabetes data with sysReq-67-team2.")
+    const acceptButton = screen.getByRole('button', { name: 'Accept' })
+    await userEvent.click(acceptButton)
+    const dialog = screen.getByRole('dialog')
+    expect(within(dialog).getByText('Add the care team sysReq-67-team2')).toBeVisible()
+    expect(within(dialog).getByText('Please enter the team code (9 digits, provided by the care team who sent this invite)')).toBeVisible()
+    const addTeamButton = within(dialog).getByRole('button', { name: 'Add team' })
+    const cancelTeamButton = within(dialog).getByRole('button', { name: 'Cancel' })
+    expect(addTeamButton).toBeDisabled()
+    await userEvent.click(cancelTeamButton)
+    expect(screen.queryByRole('dialog')).not.toBeInTheDocument()
+  })
+
+  it('Should after validation of the team code, the patient must be in a new team', async () => {
+    await act(async () => {
+      renderPage('/notifications')
+    })
+    const badgetNotification = screen.getByLabelText('Go to notifications list')
```

Comment:
badget ? what is this thing 😅
```suggestion
    const badgeNotification = screen.getByLabelText('Go to notifications list')
```